modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/loader/DefaultExtensionModelFactoryTestCase.java

```diff
@@ -389,7 +389,7 @@ private void assertSinglePrimaryContentParameter(ExtensionModel extensionModel,
   @Export(classes = {HeisenbergExtension.class, HeisenbergException.class}, resources = "methRecipe.json")
   @SubTypeMapping(baseType = Weapon.class, subTypes = {Ricin.class})
   @SubTypeMapping(baseType = Investment.class, subTypes = {CarWash.class, CarDealer.class})
-  @ErrorTypes(HeisenbergErrors.class)
+  @org.mule.sdk.api.annotation.error.ErrorTypes(HeisenbergErrors.class)
```

Comment:
I don't understand the value of this. What I would expect here is for the new annotation to use a different set of errors. Otherwise, you're not sure where the definition is coming from. As a matter of fact, I even have mixed feeling on whether this should be permitted or result in an error. I think the current tests should remain untouched, and you should use the new API to add new errors while adding new tests which assert on those.

Reply:
I just wanted to check that I don't break the model. I actually modified the errors to be SDK-api error, so they are new in a way
I should add new tests, that's correct